### PR TITLE
Added area_rectangle.py file

### PR DESCRIPTION
**Pull request by AdityaWadkar**
This script will return area of rectangle

### Describe your change:

- [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
- [x] All new Python files are placed inside an existing directory.
- [x] All filenames are in all lowercase characters with no spaces or dashes.
- [x] All functions and variable names follow Python naming conventions.
- [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
- [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
- [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by CaedenPH — CHANGES_REQUESTED** [review]
Please run the doctests with ```py
if __name__ == "__main__":
    import doctest
 
    doctest.testmod()

```

File: maths/area_rectangle.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+"""
+Basic of this concept..
```
Comment (ZeroDayOwl) [inline maths/area_rectangle.py:2]:
```suggestion
The area of a rectangle is defined as the product 
of its two adjacent sides.
```

File: maths/area_rectangle.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+https://en.wikipedia.org/wiki/Rectangle
```
Comment (ZeroDayOwl) [inline maths/area_rectangle.py:4]:
```suggestion
Source : https://en.wikipedia.org/wiki/Rectangle
```

File: maths/area_rectangle.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+"""
+
+"""area of rectangle."""
```
Comment (ZeroDayOwl) [inline maths/area_rectangle.py:8]:
```suggestion
```

File: maths/area_rectangle.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    >>> area(2,10)
+    20
+    >>> area(30,50)
+    1500
+    """
+    return length * breadth
+
+
+if __name__ == "__main__":
+    print(area(10, 20))  # --> 10,20
```
Comment (ZeroDayOwl) [inline maths/area_rectangle.py:25]:
```suggestion
```

**Review by ZeroDayOwl — CHANGES_REQUESTED** [review]

**Review by AdityaWadkar — COMMENTED** [review]
Please make a quick review  so that my code will get merged in your repository

**Comment by ZeroDayOwl** [comment]
@AdityaWadkar it seems you didn't incorporate the changes suggested. Please review them and update your algorithm.

**Pull request closed without merging**